File: app/src/main/java/com/amperas17/mangaedenapp/api/MangaApiHelper.java
package com.amperas17.mangaedenapp.api;


public class MangaApiHelper {
    public static String IMAGE_URL_PREFIX = "http://cdn.mangaeden.com/mangasimg/";
    public static int ENGLISH = 0;

    public static String buildUrl(String imageUrl) {
        return IMAGE_URL_PREFIX + imageUrl;
    }
}

File: app/src/main/java/com/amperas17/mangaedenapp/api/MangaApiHelper.kt
package com.amperas17.mangaedenapp.api


object MangaApiHelper {
    private var IMAGE_URL_PREFIX = "https://cdn.mangaeden.com/mangasimg/"
    var ENGLISH = 0

    fun buildUrl(imageUrl: String): String {
        return "$IMAGE_URL_PREFIX$imageUrl"
    }
}
